Yoast/wordpress-seo
# Take meta description date into account for the analysis

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Checks for and adds the meta description date to the description passed to the assessor for the analysis

## Test instructions

This PR can be tested by following these steps:

* In settings, make sure that Date in Snippet Preview is enabled
* Check whether the length of the date is taken into account for the analysis (e.g. have a meta description that is less than 120 characters -> the analysis should still mention that the description has a good length)

## Quality assurance


Fixes https://github.com/Yoast/wordpress-seo/issues/9985
